src/shim.js

```diff
@@ -2,6 +2,36 @@ import webpage from "webpage";
 import system from "system";
 
 /**
+ * Ensure that Function has bind() method (PhantomJS version <= 1.9 support)
```

Comment:
@akkenoth everything looks good. I just have one ask. Can you move all this to a new file? Let's create a file in a directory `polyfill/bind.js` and do `import polyfill/bind`. This file is getting big and there is no point of just adding more javascript that is not related to the actual shim.js logic.